package.json

```diff
@@ -22,7 +22,8 @@
 		"coverage:codacy": "npm run coverage:lcov && npx cross-cat coverage/lcov.info | codacy-coverage",
 		"test:js:junit": "npm run build && mocha test --reporter mocha-junit-reporter",
 		"test:js:mocha": "npm run build && mocha",
-		"test:ts:mocha": "npm run build && mocha --require ts-node/register test/test.ts"
+		"test:ts:mocha": "npm run build && mocha --require ts-node/register test/test.ts",
+		"abc": "npx chirpchirp send --message \"abc abc\n abc abc\""

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected object keys to be in natural insensitive ascending order. 'abc' should be before 'test:ts:mocha'.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5222203)